servicetalk-http-netty/build.gradle

```diff
@@ -43,16 +43,25 @@ dependencies {
   testImplementation testFixtures(project(":servicetalk-concurrent-internal"))
   testImplementation testFixtures(project(":servicetalk-http-api"))
   testImplementation testFixtures(project(":servicetalk-transport-netty-internal"))
+  testImplementation testFixtures(project(":servicetalk-http-netty"))
```

Comment:
Tests of the current module automatically depend on testFixtures from the current module, no need to have it explicitly